LanguageFeatures/Augmentation-libraries/augmenting_members_A02_t01.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It is a compile-time error if:
+/// ...
+/// A non-writable variable declaration is augmented with a setter.
+///
+/// @description Checks that it is a compile-time error if an implicitly induced
+/// getter of a final variable declaration is augmented with a setter.
```

Comment:
Note that there's nothing new about this, it is only targeting the situation where there is no setter to augment (in particular, there is no implicitly induced setter, but also no explicitly written setter).

So there's absolutely nothing in this error message other than the usual "can't augment something that doesn't exist" error.